src/runtimes.ts

```diff
@@ -62,8 +63,8 @@ export async function fromPlatform(httpClient: AxiosInstance, platformUrl: strin
 
 // Compute the runtime from the file extension.
 export function runtimeForFileExtension(fileExtension: RuntimeFileExtension): RuntimeKind | undefined {
-  const runtime = FileExtensionRuntimes[fileExtension]
-  return runtime ? `${runtime}:default` : runtime
+  const runtimes = Object.entries(FileExtensionRuntimes).filter((item, i) => item[1].includes(fileExtension))
```

Comment:
Minor point: Probably easier to use `.find()` rather than `filter()` as you only care about the first match.